task-1.js/script.js

```diff
@@ -0,0 +1,15 @@
+`use strict`;
+
+const mainTitle = document.getElementById("headerTwo");
+console.log(mainTitle.textContent);
+
+const sectionWithClass = document.querySelector(".firstSection");
+console.log(sectionWithClass.firstElementChild.textContent);
+
+const TagName = Array.from(document.getElementsByTagName("li")).filter(
+  (item) => item.textContent === "Пункт 5"
+);
```

Comment:
Nope. Take a look at the selector https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child